### PR TITLE
Added runs and cases tables

### PR DESCRIPTION
**Pull request by devalirizwan**

### CODE REVIEW

**Comment by devalirizwan** [comment]
[Add database tables: schedule_runs (Runs) and schedules (Cases)](https://trello.com/c/QV6EjsI3/688-add-database-tables-scheduleruns-runs-and-schedules-cases)

**Comment by brentkearney** [comment]
Can you please change `Runs` to `ScheduleRuns` / `schedule_runs`?

**Review by brentkearney — CHANGES_REQUESTED** [review]
Please rename the `:runs` table to `:schedule_runs`.

File: db/migrate/20211123072903_create_runs.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      t.datetime :start_time
+      t.datetime :end_time
+      t.integer :pid
+      t.date :startweek
+      t.integer :weeks
+      t.integer :runs
+      t.integer :cases
+      t.integer :aborted
+      t.integer :year
+      t.string :location
```
Comment (brentkearney) [inline db/migrate/20211123072903_create_runs.rb:13]: Perhaps this should be `t.integer :location_id` ?

**Review by brentkearney — APPROVED** [review]

**Pull request merged**